packages/pn-pa-webapp/src/pages/components/Notifications/FilterNotificationsTable.tsx

```diff
@@ -154,6 +159,18 @@ const FilterNotificationsTable = () => {
     }
   }, [filters]);
 
+  const filtersApplied = (): number =>
```

Comment:
Non dovrebbe essere necessaria questa funzione perchè la logica di filtering è stata gestita in modo diverso rispetto la persona fisica